script/gather_explorer_stake_accounts.rb

```diff
@@ -14,13 +14,16 @@ def new_epoch_detected?(network, last_epoch)
 # Run this script only if there are no ExplorerStakeAccounts for the last epoch
 if new_epoch_detected?(network, last_epoch)
   logger.info("New epoch (#{last_epoch}) detected. service started...")
-  GatherExplorerStakeAccountsService.new(
+
+  success = GatherExplorerStakeAccountsService.new(
     network: network,
     config_urls: NETWORK_URLS[network],
     current_epoch: last_epoch,
     demo: false,
     logger: logger
   ).call
+
+  CreateVoteAccountStakeHistoryService.new(network: network, epoch: last_epoch).call unless !success
```

Comment:
This is incorrect. Even if there's an error, `success` variable is true, and stat is created:

> #<ExplorerStakeAccountHistoryStat id: 5, epoch: 578, delegated_stake: 0, account_balance: 0, active_stake: 0, average_active_stake: nil, credits_observed: 0, deactivating_stake: 0, rent_exempt_reserve: 0, network: "mainnet", delegating_stake_accounts_count: 0, created_at: "2024-02-22 12:55:23.342985000 +0000", updated_at: "2024-02-22 12:55:23.342985000 +0000">

> Mysql2::Error: Duplicate entry '578-mainnet' for key 'explorer_stake_account_history_stats.index_explorer_stake_account_history_stats_on_epoch_and_network' (ActiveRecord::RecordNotUnique)